### ISSUE TITLE
Toolbar: missing database query property "tags" results in js error 

### ISSUE THREAD

**Issue by Etmolf** [underground-works/clockwork-browser#2]
Hey :)

Including the JavaScript script for the **toolbar** in a non Laravel (Vanilla) Application will result in a js error:

```
Uncaught (in promise) TypeError: e.tags is undefined
    slow https://cdn.jsdelivr.net/gh/underground-works/clockwork-browser@1/dist/toolbar.js:1
```

this is the problematic line:

https://github.com/underground-works/clockwork-browser/blob/251adb38664c9dd6336934ccf23cccd8d725a448/toolbar/request.js#L55

The problem is that the DBALDataSource registers the queries without the "tags" property:

```php
// DBALDataSource.php
// Collect an executed database query
protected function registerQuery($query)
{
	$query = [
		'query'      => $this->createRunnableQuery($query['query'], $query['params'], $query['types']),
		'bindings'   => $query['params'],
		'duration'   => (microtime(true) - $query['time']) * 1000,
		'connection' => $this->connection->getDatabase(),
		'time'       => $query['time']
	];

	if ($this->passesFilters([ $query ])) {
		$this->queries[] = $query;
	}
}
```

The Eloquent variant for Laravel does this, which is probably why there is no problem there:

```php
// EloquentDataSource.php
// Collect an executed database query
protected function registerQuery($event)
{
	$trace = StackTrace::get([ 'arguments' => $this->detectDuplicateQueries ])->resolveViewName();

	if ($this->detectDuplicateQueries) $this->detectDuplicateQuery($trace);

	$query = [
		'query'      => $this->createRunnableQuery($event->sql, $event->bindings, $event->connectionName),
		'duration'   => $event->time,
		'connection' => $event->connectionName,
		'time'       => microtime(true) - $event->time / 1000,
		'trace'      => (new Serializer)->trace($trace),
		'model'      => $this->nextQueryModel,
		'tags'       => $this->slowThreshold !== null && $event->time > $this->slowThreshold ? [ 'slow' ] : [] // <- this line is missing in the other database sources
	]
}
```

I wasn't sure if this is a problem in the clockwork-browser package or in the clockwork backend code ... probably should fix it in both places :)

Greetings

**Comment by itsgoingd** [comment]
Hey, this should indeed be handled on the side of clockwork-browser as `tags` is an optional key there.

Will get it fixed this week, thanks for the report. :)

**Comment by itsgoingd** [comment]
Well, that was a long week, but it's fixed in 1.1.1 now. (:
